src/test/java/org/opensearch/knn/index/query/InvalidSearchQueryIT.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.knn.index.query;
+
+import com.carrotsearch.randomizedtesting.annotations.ParametersFactory;
+import lombok.AllArgsConstructor;
+import lombok.SneakyThrows;
+import org.opensearch.client.Request;
+import org.opensearch.client.ResponseException;
+import org.opensearch.common.xcontent.XContentFactory;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.knn.KNNRestTestCase;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+
+import static com.carrotsearch.randomizedtesting.RandomizedTest.$;
+import static com.carrotsearch.randomizedtesting.RandomizedTest.$$;
+
+@AllArgsConstructor
+public class InvalidSearchQueryIT extends KNNRestTestCase {
+
+    private String description;
+    private XContentBuilder xContentBuilder;
+
+    @ParametersFactory(argumentFormatting = "description:%1$s; request:%2$s, expectedexception:%3$s")
+    public static Collection<Object[]> parameters() throws IOException {
+        return Arrays.asList(
+            $$(
+                $(
+                    "Empty method_parameter",
+                    XContentFactory.jsonBuilder()
```

Comment:
Could you add a comment that shows one of these in the JSON format?  It would be easier to understand the query